pkg/server/share_manager.go

```diff
@@ -188,7 +188,7 @@ func (m *ShareManager) setupDevice(vol volume.Volume, devicePath string) (string
 	}
 	m.logger.Infof("Volume %v device %v contains filesystem of format %v", vol.Name, devicePath, diskFormat)
 
-	if vol.IsEncrypted() || diskFormat == "luks" {
+	if vol.IsEncrypted() || diskFormat == "crypto_LUKS" {
```

Comment:
Good catch. So we have been using wrong format `crypto_LUKS` for a while?

Reply:
I think that `crypto_LUKS` is the right one, That's what the CSI code uses.  (And at some point, we might want to move that shared crypto code into go-common-libs.)